functions.php

```diff
@@ -7,10 +7,13 @@
 function design_system_public_enqueue_global_styles() {
     $version = filemtime( get_template_directory() . '/dist/index.css' );
     wp_enqueue_style( 'design-system-styles', get_template_directory_uri() . '/dist/index.css', array(), $version );
+    add_editor_style( 'design-system-styles', get_template_directory_uri() . '/dist/index.css', array(), $version );
```

Comment:
Do we need this? From the docs it looks like this is only for adding css to the TinyMCE editor.